.github/workflows/release.yaml

```
@@ -306,6 +310,13 @@ jobs:
           name: package-${{ matrix.platform }}
           retention-days: 1
 
+      - name: Upload web build artifact for Github Pages deployment
```

```suggestion
      - name: Upload binary to workflow artifacts for GitHub Pages (web)
```
It's interesting we only upload the binary here, and not the entire package with `index.html` etc., is that intentional?

---

It does upload the entire package with html etc in my testing, it's just we don't want the containing folder to uploaded. The regular web build is a bit nested, resulting in the page becoming available at for example TheBevyFlock.github.io/bevy_new_2d/bevy_new_2d, with the sub-directory being the repo name and the sub-sub-directory being the binary name (I think).